BootloaderCommonPkg/Include/UniversalPayload/DeviceTree.h

```diff
@@ -0,0 +1,30 @@
+/** @file
+  This file defines the structure for the PCI Root Bridges.
```

Comment:
@gdong1 this header file defines device tree for universal payload. fyi

Reply:
Thanks. it was fixed.